Makefile

```diff
@@ -1,7 +1,9 @@
 pkgs = $(shell go list ./... | grep -v /vendor/)
 files = $(shell find . -path ./vendor -prune -o -name '*.go' -print)
+ingress_binary = $(GOPATH)/bin/feed-ingress
+template = ./ingress/nginx.tmpl
 
-all : format vet lint test build
+all : format vet lint test build docker
```

Comment:
Yep. The docker task in the make file now only copies the relevant files to the docker dir. It doesn't build the image.